samples/modules/thrift/hello/doc/index.rst

```diff
@@ -0,0 +1,165 @@
+.. _thrift-hello-sample:
+
+Hello Sample Application
+########################
+
+Overview
+********
+
+This sample application includes a client a server implementing the RPC
+interface described in ``thrift/hello.thrift``. The purpose of this
+example is to demonstrate how components at different layer in thrift can
+be combined to build an application with desired features.
+
+
+Requirements
+************
+
+- QEMU Networking (described in :ref:`networking_with_qemu`)
+- Thrift dependencies installed for your host OS e.g. in Ubuntu
+
+.. code-block:: console
+   :caption: Install additional dependencies in Ubuntu
+
+   sudo apt install -y libboost-all-dev thrift-compiler libthrift-dev
```

Comment:
Not that it needs to be here for a first version, but would probably be useful to people/users if there are instructions for getting this building/working on windows too, if they can't build/run the thrift samples for the host machine itself then at least so they can build and run them on a physical device

Reply:
I think that this sort of thing will probably be addressed in a more consistent manner by the RFC below, created for the Architecture Meeting.

https://github.com/zephyrproject-rtos/zephyr/issues/54276

Recall, there were instructions for every supported platform and as part of the GSG, docs, requirements, etc, that I was asked to remove already.